src/vite-env.d.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference types="vite/client" />
+
+interface ImportMetaEnv {
```

Comment:
This makes typescript happier about `import.meta.env` variables and acts as a listing of the environment variables used in the application.